app/models/acl_dao.rb

```diff
@@ -0,0 +1,72 @@
+class AclDao
+  attr_accessor :access_token, :client_id, :base_url
+
+  def initialize(token, client_id, base_url)
+    @access_token = token
+    @client_id = client_id
+    @base_url = base_url
+  end
+
+  def get_role(user_id)
+
+    # Sets whether the user is a specific role type
+    # Note: user could be all three.
+
+    results = search_acls_by_user(user_id, 1)
+    noPages = (results['hits']/2000).ceil
+    items = Array.wrap(results['items'])
+
+    (2..noPages+1).each { |i|
```

Comment:
the brackets `{ }` here for the `each` block still need to be changed to `do` and `end`